style.md

```diff
@@ -238,16 +238,18 @@ mu.Lock()
 </td></tr>
 </tbody></table>
 
-If you use a struct by pointer, then the mutex can be a non-pointer field or,
-preferably, embedded directly into the struct.
+If you use a struct by pointer, then the mutex can be a non-pointer field.
+
+Unexported structs that use a mutex to protect fields of the struct should embed
```

Comment:
s/should/may since you may not want to make the methods part of the unexported struct's 'public' interface

```suggestion
Unexported structs that use a mutex to protect fields of the struct may embed
```